avocado/etc/avocado/.avocado.hint.example

```diff
@@ -0,0 +1,29 @@
+# This is just a example file. Insert this file inside your project root
```

Comment:
It'd only change the location of this file, and put it somewhere under `examples/`

Reply:
Fixed on v2